docs/output.md

```diff
@@ -6,15 +6,375 @@ This document describes the output produced by the pipeline. Most of the plots a
+
+Removing redundancy:
+
+- [hmmer](#hmmer-for-redundancy-removal) to match family representative sequences against other family models in order to keep non redundant ones
+- [MMseqs2](#mmseqs2-for-redundancy-removal) to strictly cluster the sequences within each of the remaining families, in order to still capture the evolutionary diversity within a family, but without keeping all the almost identical sequences
+- [FAMSA](#famsa-for-redundancy-removal) aligner option. Re-align full MSA with final set of sequences
+- [mafft](#mafft-for-redundancy-removal) aligner option. Re-align full MSA with final set of sequences
+
+Updating families:
+
+- [untar](#untar) to decompress tarballs of existing hmms and msas
+- [hmmer](#hmmer-for-updating-families) to match input sequences to existing families with hmmsearch as well as for rebuilding models with newly recruited sequences with hmmbuild
+- [SeqKit](#SeqKit) to extract fasta formatted family sequences from their MSA files
+- [MMseqs2](#mmseqs2-for-updating-families) to strictly cluster the sequences within each of the families to update
+- [FAMSA](#famsa-for-updating-families) aligner option. Re-align full MSA with final set of sequences
+- [mafft](#mafft-for-updating-families) aligner option. Re-align full MSA with final set of sequences
+- [ClipKIT](#clipkit-for-updating-families) to optionally clip gapped portions of the multiple sequence alignment (MSA)
+
+Reporting:
+
+- [Extract family representatives](#extract-family-representatives) to produce the final metadata file along with a fasta of all family representative sequences (can be used downstream for structural prediction).
 - [MultiQC](#multiqc) - Aggregate report describing results and QC from the whole pipeline
 - [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
 
+### MMseqs2
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `mmseqs/`
+  - `initial_clustering/`
+    - `mmseqs_createtsv/`
+      - `<samplename>.tsv`: tab-separated table containing 2 columns; the first one with the cluster representative sequences, and the second with the cluster members
+    - `mmseqs_createdb/`
+      - `<samplename>/`
+        - `*`: (optional) mmseqs format db of fasta sequences
+    - `mmseqs_linclust/`
+      - `<samplename>/`
+        - `*`: (optional) mmseqs format clustered db
+    - `mmseqs_cluster/`
+      - `<samplename>/`
+        - `*`: (optional) mmseqs format clustered db
+    - `filtered_fasta_chunks/`
+      - `<samplename>/`
+        - `chunked_fasta/`
+          - `*.fasta`: (optional) fasta files with amino acid sequences of each cluster above the membership threshold
+
+</details>
+
+[MMseqs2](https://github.com/soedinglab/MMseqs2) clusters amino acid fasta files via either the 'cluster' or the 'linclust' algorithms.
```

Comment:
I normally like to have one or two sentences saying waht the most useful files are (either for results intepreration, or downstream use) for each output directory

Reply:
For example, in taxprofiler: https://nf-co.re/taxprofiler/1.2.2/docs/output/#bracken